packages/twenty-front/src/modules/settings/data-model/fields/forms/components/SettingsDataModelFieldTypeSelect.tsx

```diff
@@ -116,19 +191,16 @@ export const SettingsDataModelFieldTypeSelect = ({
     <Controller
       name="type"
       control={control}
-      defaultValue={
```

Comment:
Why remove the default value?

Reply:
I fount it redundant as we are not setting any default value on the initial form load, also the user cant go to step 2 without initially selecting a field.